index.bs

```diff
@@ -379,20 +379,66 @@ If a document's [=document/embedder policy=] is "`require-corp`", then any docum
 [[#cascade-vs-require]]).
 
 To <dfn abstract-op lt="process navigation response">check a navigation response's adherence to its
-embedder's policy</dfn> given a [=response=] (|response|), and a target [=browsing context=]
-(|target|), execute the following steps, which will return "`Allowed`" or "`Blocked`" as
-appropriate:
+embedder's policy</dfn> given a [=request=] (|request|), a [=response=] (|response|), and a target
+[=browsing context=] (|target|), execute the following steps, which will return "`Allowed`" or
+"`Blocked`" as appropriate:
 
-1.  Let |response policy| be the result of [$parse header|obtaining an embedder policy$] from
+1.  Return "`Allowed`" if |target| is not a [=child browsing context=].
+
+2.  Let |response policy| be the result of [$parse header|obtaining an embedder policy$] from
     |response|.
 
-2.  Return "`Allowed`" if any of the following statements are true:
+3.  Let |parent policy| be |target|'s [=container document=]'s [=document/embedder policy=].
+
+4.  If |parent policy|'s [=embedder policy/report only value=] is "`require-corp`" and
+       |parent policy|'s [=embedder policy/report only reporting endpoint=] is not null and
+       |response policy|'s [=embedder policy/value=] is "`unsafe-none`":
+
+    1. Let |blocked url| be the first URL of |request|'s [=request/URL list=].
+
+    2. Set |blocked url|'s [=url/username=] to the empty string, and its [=url/password=] to `null`.
+
+    3. Set |serialized blocked url| be the result of executing the
+       [URL serializer](https://url.spec.whatwg.org/#concept-url-serializer) on |blocked url|
+       with the |exclude fragment flag| set.
+
+    4. Let |body| be a new object containing the following properties with keys:
+
+        * key: "`blocked`", value: |serialized blocked url|.
+        * key: "`child cross-origin-embedder-policy`", value: |response policy|'s
```

Comment:
Do we need to reveal this? I don't think there's much risk in saying that the resource asserted `same-origin` vs `same-site`, but it doesn't seem like we should be giving that information away cross-origin in the first place.

Reply:
@arturjanc to have an opinion about this (or delegate!) from the server-side.

Reply:
Ah good point. @annevk is concerned about leaking information through redirects. On the other hand, if we reveal COEP values only when the request's url is same origin with the origin of the parent context, that reveals one-bit information, same-originness. Is it too subtle?

I'm fine with revealing the COEP values only when the request's url is same origin with the origin of the parent context.

Reply:
I'm asking whether we need to reveal the value at all. "It doesn't match, otherwise I wouldn't be getting this violation report." seems like enough?

Reply:
I stated why this information is needed at https://github.com/mikewest/corpp/pull/10#issuecomment-588614040 .

Reply:
Mike, does my above comment answer your question?

Reply:
It doesn't, unfortunately. But let's land this as-is and argue about this detail offline. :)